.platform/routes.yaml

```diff
@@ -1,4 +1,5 @@
-"https://{default}/":
+"https://mercure.{default}/":
```

Comment:
Hm, if we have Mercure on a subdomain, probably we could use the default port instead of 8888?